endpoints/call.py

```diff
@@ -324,12 +326,19 @@ def handle_error(self, e):
             ret = e
 
         elif isinstance(e, TypeError):
-            # this is raised when there aren't enough args passed to controller
             exc_info = sys.exc_info()
-            logger.warning(str(e), exc_info=exc_info)
-            self.response.code = 404
-            #self.response.body = e
-            ret = e
+            tbs = traceback.extract_tb(exc_info[2])
+            filename, linenum, funcname, text = tbs[-1]
+            filename = os.path.splitext(filename)[0]
+
+            methodname = "handle_controller"
+            methodfilename = os.path.splitext(inspect.getfile(type(self)))[0]
+            if funcname == "handle_controller" and filename == methodfilename:
```

Comment:
notice what `funcname` is compared with, and then notice what is 2 lines above this line, what can you change?